src/main/java/it/pagopa/pn/deliverypush/middleware/queue/producer/abstractions/actionspool/impl/ActionsPoolImpl.java

```diff
@@ -142,125 +39,10 @@ private String computeTimeSlot(Instant instant) {
-            lastFutureActionPoolExecutionTimeDao.updateLastPollTime( instantTimeSlot );
-
-            if (toBreak)
-            {
-                // nel caso non sia riuscito a completare tutti gli timeslot, esco in modo che la prossima iterazione parta da qui.
-                log.warn("skipping remaing timeslots, last timeslotexecuted={}", timeSlot);
-                break;
-            }
-        }
-
-        Duration timeSpent = getTimeSpent(start);
-        log.debug("Action pool end. Time spent is {} millis", timeSpent.toMillis());
-
-        if(timeSpent.compareTo(lockAtMostFor) > 0){
-            log.fatal("Time spent is greater then lockAtMostFor. Multiple nodes could schedule the same actions.");
-        }
-    }
-
-    private Instant getLastPollExecuted() {
-        Optional<Instant> savedLastPollTime = lastFutureActionPoolExecutionTimeDao.getLastPollTime();
-        return getLastPollExecuted(savedLastPollTime);
-    }
-
-    private static Duration getTimeSpent(Instant start) {
-        Instant end = Instant.now();
-        return Duration.between(start, end);
-    }
-
-    private Instant getLastPollExecuted(Optional<Instant> savedLastPollTime) {
-        Instant lastPollExecuted;
-        if ( savedLastPollTime.isPresent() ) {
-            lastPollExecuted = savedLastPollTime.get();
-        } else {
-            lastPollExecuted = configs.getActionPoolEpoch();
-            if( lastPollExecuted == null ) {
-                lastPollExecuted = clock.instant().minus(2, ChronoUnit.HOURS);
-            }
-        }
-        return lastPollExecuted;
-    }
-
-    private void scheduleOne( Action action, String timeSlot) {
-        log.debug("Scheduling action {}", action );
-        
-        addToActionsQueue( action );
-        actionService.unSchedule( action, timeSlot );
-
-        log.debug("Action with actionId={} added to queue and unscheduled", action.getActionId() );
-    }
-
-    private void addToActionsQueue( Action action ) {
-        actionsQueue.push( ActionEvent.builder()
-                .header( StandardEventHeader.builder()
-                        .publisher("deliveryPush")
-                        .iun( action.getIun() )
-                        .eventId( UUID.randomUUID().toString()) // per alcuni actionId la dimensione superava gli 80 caratteri permessi per id delle code
-                        .createdAt( clock.instant() )
-                        .eventType( ActionEventType.ACTION_GENERIC.name() )
-                        .build()
-                )
-                .payload( action )
-                .build()
-        );
+    @Override
+    public void unscheduleFutureAction(String actionId) {
+        // Il metodo non fa più nulla dopo la rimozione del feature flag di performance improvement
+        // Viene lasciato per ricostruire più facilmente i punti di impatto su cui intervenire per il bug
+        // https://pagopa.atlassian.net/browse/PN-13614
     }
```

Comment:
Non c'entra nulla con lo sviluppo in questione, ma tenendo conto dell'unschedule il nuovo microservizio pn-action-manager dovrà dunque esporre anche una lambda per l'unschedule, o sbaglio?